components/fetch-build-scan-data-cmdline-tool/build.gradle.kts

```diff
@@ -1,26 +1,24 @@
-import org.hidetake.gradle.swagger.generator.GenerateSwaggerCode
```

Comment:
Global comment: please verify that the build is still configuration cache compatible.

Reply:
✅ Verified. The build is entirely configuration cache compatible.